package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "nib",
   "description": "Stylus mixins and utilities",
-  "version": "1.0.4",
+  "version": "1.0.5",
```

Comment:
this would actually be 1.1.0, since we're adding functionality, while staying backwards compatible - http://semver.org/